library/core/src/hint.rs

```diff
@@ -231,7 +231,7 @@ pub const unsafe fn assert_unchecked(cond: bool) {
 ///
 /// # Examples
 ///
-/// ```
+/// ```ignore-wasm
```

Comment:
Huh, I didn't know we supported ignore-wasm syntax in doctests... I'm not sure if we have any docs for the strings here, but seems like we should maybe add them to https://doc.rust-lang.org/rustdoc/ ?

Reply:
It is a new feature in beta: https://doc.rust-lang.org/beta/rustdoc/write-documentation/documentation-tests.html#ignoring-targets